fe/pom.xml

```diff
@@ -629,6 +629,14 @@ under the License.
                 <version>${hadoop.version}</version>
             </dependency>
 
+            <!-- https://mvnrepository.com/artifact/com.google.cloud.bigdataoss/gcs-connector -->
+            <dependency>
+                <groupId>com.google.cloud.bigdataoss</groupId>
+                <artifactId>gcs-connector</artifactId>
+                <version>hadoop3-2.2.11</version>
+                <classifier>shaded</classifier>
```

Comment:
Is this necessary?

Reply:
It's necessary, refer to https://cloud.google.com/dataproc/docs/concepts/connectors/cloud-storage#apache_maven_dependency_information